### PR TITLE
Edited license info,copyright

### PR DESCRIPTION
**Pull request by Prakharsvnit**
## Description

- Changed license and author tags.

- added copyright info.

 

## Related Issue
 #5856

 

## Motivation and Context
Fixes issue #5856

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by kdslkdsaldsal** [comment]
``<copyright>ownCloud, Inc</copyright>``

AFAIK the Inc. is not anymore.

**Comment by jvillafanez** [comment]
🚫 

I'm not aware of any change in ownCloud that support those changes. This means that either the changes proposed in the issue aren't happening, or it breaks / lose copyright and license information.
Without those changes, this PR can't be merged.

**Comment by PVince81** [comment]
@Prakharsvnit you'll need to make additional changes in the XML parsing code to make sure it properly reads the new `license-info` fields.

File: apps/comments/appinfo/info.xml [hunk 1]
```diff
@@ -3,8 +3,10 @@
 	<id>comments</id>
 	<name>Comments</name>
 	<description>Files app plugin to add comments to files</description>
-	<licence>AGPL</licence>
-	<author>Arthur Schiwon, Vincent Petry</author>
+	<license-info>
+	       <license>AGPL</license>
+	       <licensee>Arthur Schiwon, Vincent Petry</licensee>
+    </license-info>
```
Comment (PVince81) [inline apps/comments/appinfo/info.xml]: some tabbing issue

**Comment by Prakharsvnit** [comment]
@PVince81 I added xml fields to file [here](https://github.com/owncloud/core/pull/27255/files#diff-8a69640825bb5003ad9817a6d8cdd234).

File: apps/comments/appinfo/info.xml [hunk 2]
```diff
@@ -22,4 +24,5 @@
 			<collection>OCA\Comments\Dav\RootCollection</collection>
 		</collections>
 	</sabre>
+	<copyright>ownCloud, Inc</copyright>
```
Comment (PVince81) [inline apps/comments/appinfo/info.xml]: ownCloud GmbH

File: apps/updatenotification/appinfo/info.xml [hunk 3]
```diff
@@ -3,8 +3,10 @@
 	<id>updatenotification</id>
 	<name>Update notification</name>
 	<description>Displays update notifications for ownCloud and provides the SSO for the updater.</description>
-	<licence>AGPL</licence>
-	<author>Lukas Reschke</author>
+	<license-info>
+	<license>AGPL</license>
+	<licensee>Lukas Reschke</licensee>
+    </license-info>
```
Comment (PVince81) [inline apps/updatenotification/appinfo/info.xml]: indent is off

**Review by PVince81 — COMMENTED** [review]
Have you tested whether this works ?

Does the comments app in the app list properly display the license text and licensees ?

**Comment by jvillafanez** [comment]
After some discussions...
* You can ignore https://github.com/owncloud/core/issues/5856 . It's too old and it doesn't reflect the problem any longer.
* Regarding the changes from the old ticket:
    * License tag: no changes. This mean that we'll stay with the current `<license>whatever</license>`. No need to add anything for this.
    * Copyright: no longer needed. You can ignore it
    * Developers: It will be different.

As you've seen, the xml uses a "author" tag to get the authors of the apps. This is what we want to change:
* The xml will allow one or more `<author>blababla</author>` tags, each one representing **one** author. (Right now, I think it only parses one "author" tag)
* The core component parsing the xml will fetch a list of authors (be careful with compatibility) and return an array if possible.
* When the app is shown in order to be enabled or disabled, the list of authors appears there. This might need to be adjusted with the new code, and be uniform across all the apps.
* (optional) The xml tag could contain attributes. Those attributes should also be fetched.

@DeepDiver1975 If you want to add something more....

**Comment by Prakharsvnit** [comment]
@jvillafanez some doubts regarding last comment:

1)can it be like <author1>bla</author1> , <author2>bla</author2> for first point.
2)couldn't understand.
3)I think you are talking about the author names here,because that's only non-uniformity i found here.(some apps have owncloud.org in author name).

**Comment by jvillafanez** [comment]
> can it be like `<author1>bla</author1>` , `<author2>bla</author2>` for first point.

No. It has to be `<author>` for all of them.

When you enable or disable any app, the author names are shown in the web UI. I expect that info comes from the xml file, so there is likely some change to be done there to show all the authors accordingly.

There are apps with a list of authors such as "user1, user2, user3" or "user1 *and* user2" or "user1 / user2" and we can keep adding, for example "user1:user2:user3". With these new changes we aim to make this uniform.

**Comment by DeepDiver1975** [comment]
let's close this for now - there is no real need in changing this

**Pull request closed without merging**